ompi/mpiext/shortfloat/mpif-h/mpiext_shortfloat_mpifh.h.in

```diff
@@ -13,7 +15,7 @@
         integer MPIX_CXX_SHORT_FLOAT_COMPLEX
 @OMPI_MPIX_C_FLOAT16_FORTRAN_COMMENT_OUT@        integer MPIX_C_FLOAT16
 
-        parameter (MPIX_SHORT_FLOAT=74)
-        parameter (MPIX_C_SHORT_FLOAT_COMPLEX=75)
-        parameter (MPIX_CXX_SHORT_FLOAT_COMPLEX=76)
+        parameter (MPIX_SHORT_FLOAT=72)
+        parameter (MPIX_C_SHORT_FLOAT_COMPLEX=73)
+        parameter (MPIX_CXX_SHORT_FLOAT_COMPLEX=74)
```

Comment:
It would be nice to avoid having to do these.  These are scary -- i.e., are there any others hiding around the code base that we're unaware of?

Indeed, this kinda raises the question of how new datatypes should be added in MPI extensions (especially if multiple extensions can add datatypes -- they could conflict!).  Not necessarily a problem for this PR to solve, but it's one that bears some thought...